common/component/azure/servicebus/client.go

```diff
@@ -394,4 +394,4 @@ func notEqual(a, b *bool) bool {
 		return true
 	}
 	return *a != *b
-}
+}
```

Comment:
Reverting this change
```suggestion
}

```